components/sync/android/java/src/org/chromium/components/sync/BraveAndroidSyncSettings.java

```diff
@@ -6,35 +6,35 @@
 package org.chromium.components.sync;
 
 // see org.brave.bytecode.BraveAndroidSyncSettingsAdapter
-public class BraveAndroidSyncSettings extends AndroidSyncSettings {
```

Comment:
Do we need that file at all?

Reply:
This file was introduced with PR https://github.com/brave/brave-core/pull/5096. Then I had to disable it because it blocked work of Chromium syncer. I have an issue https://github.com/brave/brave-browser/issues/10417 to figure out which permissions are minimally required for sync to work, so initially I left commented code in classes and manifest.
Now I have cleaned all this.

Addressed in aafc5e184ead94535d08123a21919a61bc30c2a1

Reply:
@AlexeyBarabash I see the commit where you deleted this so why is it still showing up here as commented? https://github.com/brave/brave-core/commit/aafc5e184ead94535d08123a21919a61bc30c2a1#diff-0720b9e3bb45d0f87265000a1d304434

Reply:
@bridiver 
Sorry, my bad. I deleted `BraveAndroidSyncSettingsAdapter`  but forgot about `BraveAndroidSyncSettings`.

fixed at https://github.com/brave/brave-core/pull/5361/commits/f78f61446c5f286cb27aa2e90ff2c128b20e57de